templates/python/custom-copilot-rag-custom-api/appPackage/manifest.json.tpl

```diff
@@ -1,7 +1,14 @@
 {
+    {{#CEAEnabled}}
+    "$schema": "https://raw.githubusercontent.com/OfficeDev/microsoft-teams-app-schema/preview/DevPreview/MicrosoftTeams.schema.json",
+    "manifestVersion": "devPreview",
+    "version": "1.0.1",
```

Comment:
do we need to change the version to `1.0.1`? If not, let's change all the manifest's version to 1.0.0